modules/ram-assumable-role/variables.tf

```diff
@@ -0,0 +1,124 @@
+variable "region" {
+  description = "The region used to launch this module resources."
+  type        = string
+  default     = ""
+}
+
+variable "profile" {
+  description = "The profile name as set in the shared credentials file. If not set, it will be sourced from the ALICLOUD_PROFILE environment variable."
+  type        = string
+  default     = ""
+}
+variable "shared_credentials_file" {
+  description = "This is the path to the shared credentials file. If this is not set and a profile is specified, $HOME/.aliyun/config.json will be used."
+  type        = string
+  default     = ""
+}
+
+variable "skip_region_validation" {
+  description = "Skip static validation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
+  type        = bool
+  default     = false
+}
+
+#########################
+# alicloud_ram_role
+#########################
+variable "create_role" {
+  description = "Whether to create a role"
+  type        = bool
+  default     = false
+}
+
+variable "role_name" {
+  description = "Name of the RAM role"
```

Comment:
description = "Name of the RAM role. If not set, a default name with prefix `ram-assumable-role-` will be returned."